### PR TITLE
*: add client network metrics

### PR DESCRIPTION
**Pull request by xiang90**
/cc @heyitsanthony @gyuho

### CODE REVIEW

**Comment by gyuho** [comment]
LGTM. Thanks.

**Comment by xiang90** [comment]
@gyuho Seems like our CI is broken due to the gosimple pkg?

**Comment by gyuho** [comment]
@xiang90 Yeah... let me check

**Comment by gyuho** [comment]
Yeah `go-simple` uses custom import path, and seems like it's down now. Reported here https://github.com/dominikh/go-simple/issues/12.

Should we disable `go-simple`?

**Comment by gyuho** [comment]
it's fixed now.

File: etcdserver/api/v3rpc/metrics.go [hunk 1]
```diff
@@ -41,10 +41,27 @@ var (
 			Help:      "Bucketed histogram of processing time (s) of handled unary (non-stream) requests.",
 			Buckets:   prometheus.ExponentialBuckets(0.0005, 2, 13),
 		}, []string{"grpc_service", "grpc_method"})
+
+	sentBytes = prometheus.NewCounter(prometheus.CounterOpts{
+		Namespace: "etcd",
+		Subsystem: "network",
+		Name:      "client_grpc_sent_bytes_total",
+		Help:      "The total number of bytes sent to grpc cleints.",
```
Comment (heyitsanthony) [inline etcdserver/api/v3rpc/metrics.go]: s/cleints/clients

**Comment by heyitsanthony** [comment]
lgtm after fixing speling

**Pull request merged**